### PR TITLE
Added new tests for LPVSStatisticsService

### PR DESCRIPTION
**Pull request by v-kerimov**
# Description

Added new tests to increase test coverage.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code cleanup/refactoring
- [ ] Documentation update
- [ ] This change requires a documentation update
- [ ] CI system update
- [x] Test Coverage update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. Below is just an example and you can delete it when you contribute.

**Test Configuration**:
* Java: v11
* LPVS Release: v1.x.x

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/test/java/com/lpvs/service/scanner/scanoss/LPVSScanossDetectServiceTest.java [hunk 1]
```diff
@@ -28,41 +36,72 @@
...
-        ReflectionTestUtils.setField(scanossDetectService, "licenseService", licenseService);
-        ReflectionTestUtils.setField(scanossDetectService, "gitHubService", gitHubService);
         LPVSQueue webhookConfig = Mockito.mock(LPVSQueue.class);
         Mockito.when(LPVSWebhookUtil.getRepositoryName(webhookConfig)).thenReturn("A");
         Mockito.when(webhookConfig.getHeadCommitSHA()).thenReturn("B");
-        scanossDetectService.checkLicenses(webhookConfig);
+        Mockito.when(licenseService.findLicenseBySPDX("MIT")).thenReturn(new LPVSLicense(){{
+            setLicenseName("MIT");
+            setLicenseId(1L);
+            setSpdxId("MIT");
```
Comment (o-kopysov) [inline src/test/java/com/lpvs/service/scanner/scanoss/LPVSScanossDetectServiceTest.java:84]: This test is fixed in my PR #258 
Please exclude it from your PR.

**Review by o-kopysov — CHANGES_REQUESTED** [review]
Please check #258 and fix LPVSScanossDetectServiceTest.java accordingly.

File: src/test/java/com/lpvs/service/LPVSStatisticsServiceTest.java [hunk 2]
```diff
@@ -79,6 +94,52 @@ public void testPathCheckSendType() {
...
+            }});
+            setIssue(true);
+        }};
+        member.setNickname("testNickname");
+        when(loginCheckService.getMemberFromMemberMap(authentication)).thenReturn(member);
+        when(pullRequestRepository.findByPullRequestBase("testNickname")).thenReturn(pullRequests);
+        /*List<LPVSPullRequest> result = statisticsService.pathCheck("own", "testNickname", authentication);
+
+        assertNotNull(result);
+        assertEquals(0, result.size());*/
```
Comment (o-kopysov) [inline src/test/java/com/lpvs/service/LPVSStatisticsServiceTest.java:130]: Do we need this commented code?

**Review by o-kopysov — COMMENTED** [review]
@v-kerimov  Why did you delete new tests from LPVSScanossDetectService?

**Review by o-kopysov — APPROVED** [review]
Additional UT is added to PR #265. Approved.

**Review by tdrozdovsky — APPROVED** [review]

**Pull request merged**